pkgs/development/python-modules/cysignals/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "cysignals";
+  version = "1.6.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "003invnixqy1h4lb358vwrxykxzp15csaddkgq3pqqmswnva5908";
+  };
+
+  hardeningDisable = [
+    "fortify"
+  ];
+
+  # currently fails, probably because of formatting changes in gdb 8.0
+  doCheck = false;
+
+  preCheck = ''
+    # Make sure cysignals-CSI is in PATH
+    export PATH="$out/bin:$PATH"
+  '';
+
+  buildInputs = [
+    cython
+    sphinx
```

Comment:
If sphinx is only used for generating docs, it belongs into `nativeBuildInputs`.
Any reason for not putting `cython` in `propagatedBuildInputs`?
@FRidh What's the normal place for `cython`?